packages/components/src/index.ts

```diff
@@ -108,6 +108,7 @@ export { Heading as __experimentalHeading } from './heading';
 export { HStack as __experimentalHStack } from './h-stack';
 export { default as Icon } from './icon';
 export type { IconType } from './icon';
+export type { WPCompleter } from './autocomplete/types.ts';
```

Comment:
Hey Ramon, thank you for the ping! For future instances, using the @WordPress/gutenberg-components group handle may be better in case one of us can't reply straight away (that was me last week 😅 )

In general, we haven't really exported types from the package, so my first instinct would be to revert this change, and then consider the best course of action.

Reply:
Will do, thank you!

Reply:
Revert/update PR:

- https://github.com/WordPress/gutenberg/pull/67599